docs/manuals/source/IntroductionAndTutorial/WhatIsBareos.rst

```diff
@@ -158,6 +161,78 @@ Not all packages are required to run Bareos.
 
 -  On a Backup Administration system you need to install at least **bareos-bconsole** to have an interactive console to the |dir|.
 
+
+.. _section-BareosBinaryReleasePolicy:
+
+.. _section-BareosRepositoryTypes:
+
+Bareos Binary Release Policy
+----------------------------
+
+The Bareos project is fully committed to Open-Source.
```

Comment:
```suggestion
The Bareos project is fully committed to open source.
```

Reply:
Replaced by open-source (as found in https://en.wikipedia.org/wiki/Open_source)